src/main/java/com/itschool/eventmanagment/services/EventServiceImp.java

```diff
@@ -63,7 +59,28 @@ public List<EventDTO> getEvents(LocalDateTime to, LocalDateTime from, String loc
         } else {
             events = eventRepository.findByEventDateTimeBetweenAndLocationContainingIgnoreCaseOrderByEventDateTime(to, from, location);
         }
-        return events.stream().map(event -> objectMapper.convertValue(event, EventDTO.class)).collect(Collectors.toList());
+        return events.stream().map(EventServiceImp::mapEventToEventDto).toList();
+    }
+
+    private static EventDTO mapEventToEventDto(Event event) {
```

Comment:
Since we have this mapping method private, it doesn't make sense to use `static`. At this moment, and maybe in the future, this method will be used here in this service. If you really want to make use of this method in the future in other classes/services, you can create a `utils` package on the same level as `controllers`, `services` and design a class with this specific purpose that can have static methods for converting DTOs to entities and the other way around.

Reply:
@AlexandruIliescu Thanks. Since this PR has deviated from its initial purpose and new changes have been requested each time despite the back-and-forth, I will create a new PR on the dev branch to be resolved against the main branch, in order to address these changes in an organized manner.